srcpkgs/clifm/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'clifm'
+pkgname=clifm
+version=1.11
+revision=1
+build_style=gnu-makefile
+make_install_args="MANDIR=/usr/share/man"
+makedepends="libcap-devel acl-devel file-devel readline-devel"
+depends="libmagic"
+short_desc="Clifm is a Command Line Interface File Manager"
+maintainer="Greg Beard <gmbeard@googlemail.com>"
+license="GPL-2.0-only"
+homepage="https://github.com/leo-arch/clifm"
+changelog="https://raw.githubusercontent.com/leo-arch/clifm/v${version}/CHANGELOG"
```

Comment:
```suggestion
changelog="https://raw.githubusercontent.com/leo-arch/clifm/master/CHANGELOG"
```
don't set this to the tag so users/maintainers can see the entire changelog, not just up to the tag

Reply:
Fixed. Thank you for the review